src/Augmentation/ui/AugmentationsRoot.tsx

```diff
@@ -72,11 +75,34 @@ export function AugmentationsRoot(props: IProps): React.ReactElement {
       </Typography>
       <Box mx={2}>
         <Tooltip title={<Typography>'I never asked for this'</Typography>}>
-          <span>
-            <Button disabled={player.queuedAugmentations.length === 0} onClick={props.installAugmentationsFn}>
+          <>
+            <Button disabled={player.queuedAugmentations.length === 0} onClick={() => setInstallationOpen(true)}>
               Install Augmentations
             </Button>
-          </span>
+            <Modal open={isInstallationOpen} onClose={() => setInstallationOpen(false)}>
+              <Typography>
+                WARNING: THIS WILL PARTIALLY RESET YOUR PROGRESS
+                <br />
+                <br />
+                For your first installation, it is recommended to purchase most of the CSEC augmentations.
+                <br />
+                If you are unsure, you should EXPORT your save game as a backup before!
+                <br />
+                <br />
+                <Typography variant="subtitle2"><em>This is the first prestige mecanism.</em></Typography>
```

Comment:
```suggestion
                <Typography variant="subtitle2"><em>This is the first prestige mechanism.</em></Typography>
```
Should this even be mentioned at all?

Reply:
Possibly not but some player seemed confused by the apparent progress reset. I can remove it if preferred.

Reply:
I meant the hint that there was more than one prestige mechanism.